test/powershell/engine/Help/assets/HelpURI/V3Cmdlets.csv

```diff
@@ -1,98 +1,98 @@
 TopicTitle,HelpURI
 Add-ContentFileSystem,https://go.microsoft.com/fwlink/?LinkID=204573
 Add-JobTrigger,https://go.microsoft.com/fwlink/?LinkID=223913
-Connect-PSSession,https://go.microsoft.com/fwlink/?LinkID=210604
-ConvertFrom-Json,https://go.microsoft.com/fwlink/?LinkID=217031
-ConvertTo-Json,https://go.microsoft.com/fwlink/?LinkID=217032
+Add-LocalGroupMember,https://go.microsoft.com/fwlink/?LinkId=717987
+Clear-RecycleBin,https://go.microsoft.com/fwlink/?LinkId=2109377
+Compress-Archive,https://go.microsoft.com/fwlink/?linkid=2096473
```

Comment:
Please remove this change till the module has shipped with the update.

Reply:
Removed `Compress-Archive` from the CSV file.